spec/models/piece_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'rails_helper'
+
+RSpec.describe Piece, type: :model do
+  describe 'is_obstructed?' do
+    it 'should return false for no horizontal obstructions moving to the right' do
```

Comment:
I'll usually leave out `should`

`it 'returns false...'` works, less typing :)